Program.fs

```diff
@@ -24,6 +23,11 @@ let webApp =
             choose [
                 route "/" >=> razorHtmlView "Index" { Text = "Hello world, from Giraffe!" }
             ]
+        POST >=>
```

Comment:
I wonder if it would make more sense to flip the routes so that it would be:

```
route "/receipts" >=>
  choose [
    GET >=> getHandler;
    POST >=> postHandler;
  ]
```